content/en/containers/guide/kubernetes-cluster-name-detection.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Kubernetes cluster name auto detection
+title: Kubernetes cluster name automatic detection
```

Comment:
```suggestion
title: Kubernetes Cluster Name Automatic Detection
```
Recommend title case unless there is some style you are consistently using here that I'm not aware of.